src/common/snippets/src/pass/lowered/loop_init.cpp

```diff
@@ -0,0 +1,234 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/pass/lowered/loop_init.hpp"
+#include "snippets/snippets_isa.hpp"
+#include "snippets/itt.hpp"
+
+namespace ngraph {
+namespace snippets {
+namespace pass {
+namespace lowered {
+
+namespace {
+void get_io_exprs(LoweredExprIR& linear_ir,
+                  const std::vector<LoweredExprPort>& loop_entries, const std::vector<LoweredExprPort>& loop_exits,
+                  std::vector<LoweredExprPtr>& loop_in_exprs, std::vector<LoweredExprPtr>& loop_out_exprs,
+                  OutputVector& loop_in_outputs, OutputVector& loop_out_outputs) {
+    loop_in_exprs.clear();
+    loop_out_exprs.clear();
+    loop_in_outputs.clear();
```

Comment:
We probably don't need to carry OutputVector around, since we don't use it anymore. May be fixed in leftovers, if we don't have time.